_posts/2023-04-05-deploying-eks-within-restricted-private-subnets.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Deploying EKS within Restricted Private Subnets"
+date:   2023-04-05 11:16:50 +0530
+categories: Test Automation
+picture: "../images/blog1/img2.png"
```

Comment:
This image URI is incorrect. Kindly update it to blog5 instead of blog1.

Reply:
updated